### PR TITLE
Styled Player controls, replacing icons with SVG/PNG assets

### PR DESCRIPTION
**Pull request by BC-Mikey**
This pull request has updated SVG & PNG stylesheets as well as styled player controls which use these assets.

### CODE REVIEW

File: img/svgcss/icons.data.png.css [hunk 1]
```diff
@@ -39,17 +39,63 @@
...
 	background-repeat: no-repeat;
 }
 
-.bcc-img-share {
-	background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABEAAAAWCAYAAAAmaHdCAAAAzUlEQVQ4T+2UYRHCMAyFiYNJmARwMAmTAkpAAg7AATgACaAkvFdSbt2atvzf7nJd95Kv6Wtvsik8qtpBviAOIvL0UsUTANgaoMf4RuwBuubysxADsGBEPBA7xN1A5zloAQGAK7Ng5BYwV4xiW+P3E6YJyOukR+KLK0aIvdOjLmqxI9eTmDCFuMYi6eaI4UQm26HRx6yxtZWa9KYktFS4CkrXg/vFpJq+QhL7gh0Nngzwldc9+zRBvOLkRtc6WSE1B5b673TsT/Y/4VsxfABYMdGXBx544gAAAABJRU5ErkJggg==');
+.bcc-player-button.bcc-social-control:hover:before {
+	background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABEAAAAWCAYAAAAmaHdCAAAA9klEQVQ4T+2UTQrCMBCFZ6qgCxc9QpeCChY9gEfwGtpNPUnd2G69gb2BPYASoQou9SDN2FEi/iRtQXDVrAJv8s3MmyQIBcsJhN1pZRuixuLkuQdTKJqE3koMLZSbXHeI4IpIfjofx7p4LYQBCDImsKY5SEiyXAuzBAj91ButP0FfkG4knKaUCQO4hUG4p3Q+wkdrMkGC5SdIWwmDzjP3whkVhPcMarfBVpqqyOiJCniFGI3th7utTlQTUZC7T5gFWmPLMlXRsUoQG2tqhc/XkHd3/udJPxKT48xNfppO0VehnkXpiGtImQPf+vOeEJBxhGVYBJzcAEtz0sXNXngrAAAAAElFTkSuQmCC');
```
Comment (bceacuna) [inline img/svgcss/icons.data.png.css]: We shouldn't change the generated code manually. We might need to streamline this  process  of generating the image sprite in the future.  If it is possible, try to keep the svg  unaltered (not a big deal)

**Comment by bceacuna** [comment]
Looks good. Just a comment that can be ignored.

File: img/svgcss/icons.data.png.css (hunk 1 above)
Comment (BC-Mikey) [inline img/svgcss/icons.data.png.css]: Are you referring to the class names I used?

Comment (bceacuna) [inline img/svgcss/icons.data.png.css]: yes the class names (i.e bcc-player-button.bcc-social-control:hover:before)

**Pull request merged**